assets/src/dashboard/app/font/fontProvider.js

```diff
@@ -54,7 +54,15 @@ function FontProvider({ children }) {
   );
 
   useEffect(() => {
-    getAllFonts().then(setFonts);
+    let mounted = true;
+    getAllFonts().then((newFonts) => {
+      if (mounted) {
+        setFonts(newFonts);
+      }
+    });
+    return () => {
+      mounted = false;
+    };
```

Comment:
We should be able to cancel promise like we clear timeouts or intervals 😞 But nice workaround!